docs/network-flow-visibility.md

```diff
@@ -109,7 +112,7 @@ Please modify them as per your requirements.
 
```

Comment:
L106: Please replace `flowExportFrequency` with `activeFlowExportTimeout` and `idleFlowExportTimeout `

Reply:
Thanks, addressed.